extensions/wikia/AdEngine3/src/tracking/tracker.js

```diff
@@ -19,6 +19,7 @@ export const track = (data) => {
 };
 
 export const registerSlotTracker = () => {
+	slotTracker.onChangeStatusToTrack.push('catlapsed');
```

Comment:
Every-time we register slot tracker we push `'catlapsed'` -- it doesn't sound right. `registerSlotTracker` should just register the `slotTracker` it shouldn't extend the statuses to track and modify `slotTracker`. What stops us from just modifying `slotTracker` in the AdEngine?

Reply:
Are we sure we want to move that to AdEngine? This is kind of an edge case within app and m-w. Those are the only platforms that use that and i'm not sure if we want to use that in other platforms.